fe/fe-core/src/main/java/com/starrocks/catalog/Database.java

```diff
@@ -137,7 +148,14 @@ public void readUnlock() {
     }
 
     public void writeLock() {
+        long startMs = System.nanoTime() / 1000000;
         this.rwLock.writeLock().lock();
+        long endMs = System.nanoTime() / 1000000;
+        if (endMs - startMs > SLOW_LOCK_MS && endMs > lastSlowLockLogTime + SLOW_LOCK_LOG_EVERY_MS) {
+            lastSlowLockLogTime = endMs;
+            LOG.warn("slow write lock db:" + id + " " + fullQualifiedName + " " + (endMs - startMs) + "ms",
```

Comment:
log in the lock may cause the lock time to be longer?

Reply:
3-second log one is not a very serious problem I think